### ISSUE TITLE
wrong ubuntu version number

### ISSUE THREAD

**Issue by snickers2k** [webmin/webmin#1088]
```
root@cloudBox:~# cat /etc/issue
Ubuntu 18.04.3 LTS \n \l
```

Webmin still has

`Operating system | Ubuntu Linux 16.04.5`

**Comment by rostovtsev** [comment]
Hi,

Don't you have a prompt on Dashboard offering to update upgraded system?

**Comment by sz00gun** [comment]
Try to go to webmin configuration, then to Operating system and Environmental

You can detect you version there and easily change it as well there.

I have a problem with automatic version change as well. Every so often I change or manually... (Version info)

**Comment by rostovtsev** [comment]
Yes, correct, although it should provide a prompt on Dashboard for updating it automatically.

**Comment by sz00gun** [comment]
Never seen that before. Either on Ubuntu, Debian or Centos, in the last 7 years

**Comment by rostovtsev** [comment]
@jcameron Maybe it's _Debian_  related bug, because I see it quite regularly on _RHEL_  systems?

**Comment by snickers2k** [comment]
<img width="907" alt="Bildschirmfoto 2019-08-13 um 11 30 00" src="https://user-images.githubusercontent.com/5986024/62930991-d1552e80-bdbd-11e9-86b9-bd8a990f5cf1.png">

No, there was no prompt in Dashboard, OS is up2date.
Changed it manually.

**Comment by sz00gun** [comment]
That's the way I do it. Never seen it done automatic.
